pandas/core/window/rolling.py

```diff
@@ -1901,7 +1901,7 @@ def _get_corr(a, b):
                 window=window, min_periods=self.min_periods, center=self.center
             )
 
-            return a.cov(b, **kwargs) / (a.std(**kwargs) * b.std(**kwargs))
+            return a.cov(b, **kwargs) / (a.var(**kwargs) * b.var(**kwargs)) ** 0.5
```

Comment:
can you add some comments here (and ref this issue)

Reply:
Done